Makefile

```diff
@@ -251,6 +251,7 @@ flake8: requirements .flake8
 	@echo
 	@echo "==================== flake ===================="
 	@echo
+	. $(VIRTUALENV_DIR)/bin/activate; cd ./lint-configs/python/flake8_plugins; python setup.py develop
```

Comment:
That works for now although this usually results in a lot of issues if package is updated or similar so if we will have issues with it, we should move it to a separate repo and install it from git using using pip.

Reply:
@Kami Yeah, this is temporary until we setup the flake8 plugins repo and upload the package into pypi.